### ISSUE TITLE
Standards sweep: trait NonFungibleTokenCore contains non-core methods

### ISSUE THREAD

**Issue by mikedotexe** [near-apps/nft-market#11]
Here:
https://github.com/near-apps/nft-market/blob/4fb70e6a4f9b8db6fd51ccbc674dcfdcb2e1ba57/contracts/nft-simple/src/nft_core.rs#L10

We have methods that belong in approval and enumeration and payout traits instead.

**Comment by mattlockyer** [comment]
Aren't traits implementation details? Isn't standard just the interface of the contract and method signatures?

**Comment by austinabell** [comment]
I think the point may be that it's just a bit confusing, and it doesn't have to redefine the interface? Sorry Mike if I'm putting words in your mouth here

**Comment by mattlockyer** [comment]
Yea I get it, so just break up the methods by trait as defined in the standards?

**Comment by austinabell** [comment]
yeah, I'll play around with it in a minute. I'm not 100% sure what I'm thinking is doable with how the standards API is currently setup
